client/my-sites/customer-home/cards/tasks/task.jsx

```diff
@@ -36,8 +37,12 @@ import './style.scss';
 const Task = ( {
 	actionText,
 	actionUrl,
+	actionCallback,
+	chipText,
 	description,
 	illustration,
+	dismissable,
+	skippable = true,
```

Comment:
Optional: To avoid confusion (both skippable and dismissable could mean the same thing) can we maybe replace them with a `areSkipOptionsEnabled` prop (which defaults to `true`)?

We can then reuse the same skip button but changing its content/behavior depending on that property:

```
<Button
	className="task__skip is-link"
	ref={ skipButtonRef }
	onClick={ () => areSkipOptionsEnabled ? setSkipOptionsVisible( true ) : skipTask( 'never' ) }
>
		{ areSkipOptionsEnabled ? translate( 'Remind me' ) : translate( 'Dismiss' ) }
		{ areSkipOptionsEnabled && <Gridicon icon="dropdown" size={ 18 } /> }
</Button>
{ areSkipOptionsEnabled && areSkipOptionsVisible && (
		<PopoverMenu> ... </PopoverMenu>
) }
```

Reply:
Nice 👍 I like this approach much better, so switched to this.